data/sources/blocks/mci/change-comment-block.yaml

```diff
@@ -0,0 +1,38 @@
+id: change-comment-block
+questions:
+- answers:
+  - guidance:
+      content:
+      - description: 'Examples of commentary:'
+      - title: "\u2018In-store promotion\u2019"
+      - description: "\u201COffer on wine for the whole month (save 25% when you buy\
+          \ 6 bottles). The promotion was available in-store and online, contributing\
+          \ to an increase in both total retail turnover and internet sales.\u201D"
+      - title: "\u2018Special events (for example, sporting events)\u2019"
+      - description: "\u201CThis was the month before the start of Euro 2016 (football),\
+          \ we recorded an increase in sales of audio-visual equipment (for example,\
+          \ televisions and audio equipment). This led to an increase in sales both\
+          \ in-store and online.\u201D"
+      - title: "\u2018Weather\u2019"
+      - description: "\u201CThe bad weather has decreased our sales of summer clothing.\
+          \ This has led to a reduction in total retail turnover and internet sales\
+          \ this month.\u201D"
```

Comment:
Not the library, this is how the JSON is currently https://github.com/ONSdigital/eq-survey-runner/blob/master/data/en/1_0205refresh.json#L617